sitback-furniture/src/components/cart-navbar/CartNavbar.module.scss

```diff
@@ -0,0 +1,24 @@
+@import "../../styles/colors";
+@import "../../styles/units";
+
+.cartNavbar {
+    display: flex;
+
+    .navItem {
+        color: $black;
+        text-decoration: none;
+        text-transform: uppercase;
+        font-size: 16px;
+        font-weight: 500;
+        margin: 0 $unit-7;
+        cursor: pointer;
+
+        &:not(.active) {
+            color: #888888;
```

Comment:
could you let me know if the color codes be moved to _variables.scss

Reply:
Moved all colors to scss variables